docs/node/run-your-node/keymanager-node/signing-key-manager-policy.md

```diff
@@ -0,0 +1,192 @@
+# Signing Key Manager Policy
+
+This guide will describe how to print and sign an Oasis key manager policy.
+
+:::info
+
+These instructions are only applicable if you are part of a key manager policy
+signer set.
+
+:::
+
+## Prerequisites
+
+### Oasis Node Binary
+
+Make sure you have followed the [Oasis Node binary installation guide] and have
+the Oasis Node binary installed on your system.
+
+### Entity
+
+Similarly to other things, an entity's private key is used to sign a key manager
+policy.
+
+The trusted key manager policy signer set is hard-coded in the key manager's
+source code.
+The trusted signer set for the Oasis Key Manager is defined in [its source code
+][oasis-km-signer-set].
+
+:::danger
+
+We strongly recommend using a dedicated (single-purpose) entity for signing key
+manager policies for production key managers, i.e. the ones deployed on
+Mainnet and connected to a production ParaTime.
+
+:::
+
+
+To provision a new entity, follow the [instructions in our Validator Node
+guide].
+
+:::danger
+
+We strongly advise **against** using an entity that is generated with a
+file-based signer for singing key manager policies for production key manager,
+i.e. the ones deployed on Mainnet and connected to a production ParaTime.
+
+:::
+
+:::danger
+
+In case a file-based signer needs to be used, we strongly recommend using an
+[offline/air-gapped machine] for this purpose and never exposing the entity's
+private key to an online machine.
+
+Gaining access to the entity's private key can compromise the trusted key
+manager policy signer set and hence the key manager itself.
+
+:::
+
+:::info
+
+Currently, Ledger-based signers do not support signing key manager policies.
+
+:::
+
```

Comment:
```suggestion

[key manager policy]: https://github.com/oasisprotocol/oasis-core/blob/master/docs/consensus/services/keymanager.md#policies
```

FYI linking to `master` branch above is fine - the cross-repo-link plugin will rewrite it to the local URL.